catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/geofeature/GazetteerFeatureService.java

```diff
@@ -69,22 +95,36 @@ public Feature getFeatureByName(String name) {
     return null;
   }
 
-  protected GeoResult getGeoResultFromGeoEntry(GeoEntry entry) {
-    return GeoResultCreator.createGeoResult(entry);
+  private String getAlpha3CountryCodeForGeoEntry(GeoEntry entry) {
+    String featureCode = entry.getFeatureCode().toUpperCase();
+    if (!featureCode.startsWith("PCL")) return null;
```

Comment:
Does the formatter allow this?  I think generally we would use `{}`s for this conditional